PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -110,20 +130,22 @@ fslmaths "$OutputMNIT1wImage" -abs "$OutputMNIT1wImage" -odt float
 fslmaths "$OutputMNIT1wImage" -div "$BiasFieldOutput" "$OutputMNIT1wImageRestore"
 fslmaths "$OutputMNIT1wImageRestore" -mas "$T1wMNIImageBrainMask" "$OutputMNIT1wImageRestoreBrain"
 
-applywarp --rel --interp=spline -i "$OrginalT2wImage" -r "$T1wImageBrainMask" -w "$OutputOrigT2wToStandard" -o "$OutputMNIT2wImage"
-fslmaths "$OutputMNIT2wImage" -abs "$OutputMNIT2wImage" -odt float
-fslmaths "$OutputMNIT2wImage" -div "$BiasFieldOutput" "$OutputMNIT2wImageRestore"
-fslmaths "$OutputMNIT2wImageRestore" -mas "$T1wMNIImageBrainMask" "$OutputMNIT2wImageRestoreBrain"
+if [ "${T2wPresent}" = "YES" ] ; then
+  applywarp --rel --interp=spline -i "$OrginalT2wImage" -r "$T1wImageBrainMask" -w "$OutputOrigT2wToStandard" -o "$OutputMNIT2wImage"
+  fslmaths "$OutputMNIT2wImage" -abs "$OutputMNIT2wImage" -odt float
+  fslmaths "$OutputMNIT2wImage" -div "$BiasFieldOutput" "$OutputMNIT2wImageRestore"
+  fslmaths "$OutputMNIT2wImageRestore" -mas "$T1wMNIImageBrainMask" "$OutputMNIT2wImageRestoreBrain"
 
```

Comment:
Ok, this is being picky, but for symmetry with the preceding, I'd conclude the preceding conditional with a `fi` here, and start a new `if [ "${T2wPresent}" = "YES" ] ;` in the `#Create T1w/T2w maps` section, so that the if conditional doesn't "bleed" across the distinct code sections in a weird fashion.

Reply:
done